code/modules/jobs/job_types/roguetown/nobility/lord.dm

```diff
@@ -28,7 +28,7 @@ GLOBAL_LIST_EMPTY(lord_titles)
 	whitelist_req = FALSE
 	min_pq = 10
 	max_pq = null
-	give_bank_account = 1000
+	give_bank_account = TRUE
```

Comment:
Regarding this, currently, for some reason most times this var appears I'm jobs it has an arbitrary value. Since you're fixed that here please do so with all other jobs too since you're already touching this system please!

Reply:
look at code/modules/jobs/job_types/_job.dm 188 line

Reply:
So you're telling me account have their own number and aren't directly tied to the player?
If so setting it to TRUE or FALSE would be entirely the same as TRUE is 1 and that checks for numbers superior to 1. I need to check this again when I'm back home.